glossary.rst

```diff
@@ -84,5 +62,35 @@ Glossary
         .. note::
             If :term:`analytics attribute` uses the same name - it will be represented as a separate :term:`attribute`.
 
-    Analytics
-        PPAS component gathering statistics about each :term:`user` of the :term:`application` (previously **Piwik**).
+    Device ID
+        Device ID (device identification) is a distinctive number associated with a smartphone or similar handheld
+        device. Device IDs are separate from hardware serial numbers.
+
+    Identifier
+        Unique identifier of a :term:`visitor` ID (e.g. :term:`analytics ID`, :term:`user ID`, :term:`device ID` or email).
+
+    JavaScript Tracking Client (JSTC)
+        A JavaScript object that is able to send requests to CPP. It is loaded and created with download of `ppms.js` file. It has an :ref:`API<data-collection-javascript-tracking-client-api>` that allows to configure what data requests should contain. You can :ref:`learn more about JSTC here<data-collection-javascript-tracking-client-installation>`
+
+    JavaScript Tracking Snippet (JSTS)
+        A JavaScript code, usually in form of HTML tag, that initiates JSTC and sends first tracking request. You can see an :ref:`example of JSTS here<jtc-installation-jsts-example>`.
+
+    PII
+        Personally Identifiable Information.
+
+    Tracking Tag
```

Comment:
I'm not sure this is a term. Term should be used in multiple places and referenced constantly without explaining what it is (that's where link to this description comes in).

Reply:
I did not add it, cleaning up the glossary can be another task. If you feel so, please fill one.